front/pages/api/w/[wId]/assistant/agent_configurations/index.ts

```diff
@@ -10,13 +11,14 @@ import {
   assertNever,
   Err,
   GetAgentConfigurationsQuerySchema,
+  isDustAppRunConfiguration,
   Ok,
   PostOrPatchAgentConfigurationRequestBodySchema,
   removeNulls,
 } from "@dust-tt/types";
 import { isLeft } from "fp-ts/lib/Either";
 import * as reporter from "io-ts-reporters";
-import _ from "lodash";
+import _, { uniq } from "lodash";
```

Comment:
`nit` can we either import Lodash functions individually or import the entire library?